### ISSUE TITLE
participant feedback on lesson material from ENCCS Feb 2022 workshop

### ISSUE THREAD

**Issue by wikfeldt** [carpentries-incubator/deep-learning-intro#201]
just thought i'd copy-paste this here, these are the answers to our post-workshop survey questions that relate to the material (see #188 ). Nice to see that most participants found the material excellent

### Q: What did you like best about the lesson material, exercises and teaching? Where should we improve?

- Nice setup with Hackmd and Course material to easily copy-paste code snippets to be able to follow along. Day two was a bit fast paced for me. Maybe a bit slower and a bit more explanation on what the different improvements/changes to the model does. Also maybe more explanation, what loss and accuracy actually tell me.
- That was a very well prepared and balanced material.
- Good material, intuitive hands-on exercises. More visual aids of the networks and data could be useful.
- The lesson material run without issues. Perhaps the notebooks could be ready made and tutors write comment into the notebook while explaining. Then there will be no copy-pasting and instead focus on explanation - function and code. Perhaps make a tutorial on pandas dataframes as workshop preparation would be good.
- Easily available material, all the commands are ready to copy and paste and well explained. Maybe I would add sensitivity, specificity to the course topics, or what changes in the model performances if the number of epochs/layers/neurons is changed .
- Hello, I think maybe it is better to have hands-on parts. In this way. one cal deal with in details to what learn from workshop.
- + very practical - copy pasting
- it's ok
- content and materials teaching. maybe a common platform for jupyter notebook to have a live update correction.
- I like how well explained was everything. I think I was finally able to understand some difficult concepts in deep learning.
- Everything was well explained and documented. Enough time for questions, explanations and examples
- examples provided and hands-on training including the breakout room were good.
- The material was excellent! the teaching and discussions clear and interesting. I believe there is little to improve. There is something, but don't know what, that could be done to incentivize student participation. But all in all it was better than other online courses of the kind I have attended.

## Q: What topic did you find most interesting, relevant or useful?

- I like the weather prediction. It would have been cool to have more time/opportunity/discussion/trials to improve one model instead of doing the third example.
- All, good sequence of material
- Typical defaults and ways to improve model performance.
- I would have liked even more focus on how the various layers in a network are introduced and applied to various problems.
- Improving the model by reduction of overfitting
- All topics were so useful, according to my idea.
- depends on your own ML project I guess so maybe ask this in advance? For me the weather data was most interesting
- cnn.
- All of them.
- Introduction and classifiers were relevant, similarly all the discussion about the problem of overfitting
- Design and train a Deep Neural Network Troubleshoot the learning process
- The topics on the last day, Following model training and CNNs.

## Q: What topic did you find least interesting, relevant or useful?

- image classification, but also I was a bit lost there
- For y specific applications all was useful
- None of them.
- Conv networks, because I'm not working with image classifications (see above, depends on your own project)
- classification
- The part on convolutional neural network, which I find more distant from my field of research
- I don't think that any.

**Comment by dafnevk** [comment]
Thanks a lot for sharing!

**Comment by dafnevk** [comment]
No further actionable points, so I will close this issue.
